src/blocks/events/styles/editor.scss

```diff
@@ -0,0 +1,61 @@
+.wp-block[data-type="coblocks/events"] {
+	.coblocks-list-appender {
+		margin-left: -12px;
+		margin-right: -12px;
+		margin-top: 28px;
+	}
+
+	.coblocks-list-appender__toggle {
```

Comment:
In the future we should probably pull this out and create a component (as we use this in a couple other blocks as well).

Reply:
I only left it in because the code is so small right now.